src/QuickMarcEditor/utils.js

```diff
@@ -807,3 +807,37 @@ export const splitFields = marcRecord => {
     }),
   };
 };
+
+export const are010Or1xxUpdated = (initial, updated) => {
+  let is010Updated = false;
+  let is1XXUpdated = false;
+
+  const initial010 = initial.find(rec => rec.tag === '010');
```

Comment:
Move `010` to const, please

Reply:
Constant can be called `CONTROL_NUMBER_FIELD`